lib/ansible/module_utils/gandi_livedns_api.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019 Gregory Thiemonge <gregory.thiemonge@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
code in module_utils must not be GPL licensed.

Reply:
fixed in last push